src/app/static/src/tests/filteredData/getters.test.ts

```diff
@@ -126,4 +124,258 @@ describe("FilteredData mutations", () => {
+                indicator: "artcov",
+                est: 5
+            }
+        ];
+        const testRootState = {
+            version: "",
+            selectedDataType: null,
+            baseline: {...initialBaselineState},
+            surveyAndProgram: mockSurveyAndProgramState(
+                {survey: mockSurveyResponse(
+                        {data: testData}
+                    )}),
+            filteredData: testState
+        };
+
+        const regionIndicators = getters.regionIndicators(testState, getters, testRootState, null);
+
+        const expected = {
+            indicators: {
+                "area1": {"prev": 2},
+                "area2": {"prev": 3, "art": 5},
+                "area3": {"art": 4}
+            },
+            artRange: {min: 4, max: 5},
+            prevRange: {min: 2, max: 3}
+        };
+
+        expect(regionIndicators).toStrictEqual(expected);
+    });
+
+    it("gets regionIndicators for programme", () => {
+        const testStore:  Module<FilteredDataState, RootState> = {
+            state: {...initialFilteredDataState, selectedDataType: DataType.Program},
+            getters: getters
+        };
+        const testState = testStore.state as FilteredDataState;
+        const testData = [
+            {
+                iso3: "MWI",
+                area_id: "area1",
+                current_art: 2
+            },
+            {
+                iso3: "MWI",
+                area_id: "area2",
+                current_art: 3
+            }
+        ];
+        const testRootState = {
+            version: "",
+            selectedDataType: null,
+            baseline: {...initialBaselineState},
+            surveyAndProgram: mockSurveyAndProgramState(
+                {program: mockProgramResponse(
+                        {data: testData}
+                    )}),
+            filteredData: testState
+        };
+
+        const regionIndicators = getters.regionIndicators(testState, getters, testRootState, null);
+
+        const expected = {
+            indicators: {
+                "area1": {"prev": 2},
+                "area2": {"prev": 3},
+            },
+            artRange: {min: null, max: null},
+            prevRange: {min: 2, max: 3}
+        };
+
+        expect(regionIndicators).toStrictEqual(expected);
+    });
+
+    it("gets regionIndicators for ANC", () => {
+        const testStore:  Module<FilteredDataState, RootState> = {
+            state: {...initialFilteredDataState, selectedDataType: DataType.ANC},
+            getters: getters
+        };
+        const testState = testStore.state as FilteredDataState;
+        const testData = [
+            {
+                iso3: "MWI",
+                area_id: "area1",
+                ancrt_test_pos: 2
+            },
+            {
+                iso3: "MWI",
+                area_id: "area2",
+                ancrt_test_pos: 3
+            }
+        ];
+        const testRootState = {
```

Comment:
This testState object is mostly the same each time, I've added a helper in another branch that can tidy this up a bit, might be nice to bring it in here:
https://github.com/mrc-ide/hint/pull/56/files#diff-06da03af1436a7e713350bccd618bf8bR61